docs/source/trainer/numerics.md

```diff
@@ -0,0 +1,73 @@
+# 🔢 Numerics
+
+Composer supports several single and half-precision number formats including IEEE single-precision `fp32`, IEEE half-precision `fp16`, Google’s truncated `bf16` format and support for mixed precision training `amp`.  The following formats are supported per accelerator:
+
+| Format | CPU | GPU - A100 | GPU - V100 | GPU - T4 |
+| --- | --- | --- | --- | --- |
+| bf16 | ➖ | ✅ | ➖ | ➖ |
+| fp16 | ➖ | ✅ | ✅ | ✅ |
+| fp32 (default) | ✅ | ✅ | ✅ | ✅ |
+| amp | ➖ | ✅ | ✅ | ✅ |
+
+When using the {class}`~.Trainer`, the number format can selected by specifying the `precision` argument during initialization. In the example below, we are training on a `gpu` device and using the `fp16` format:
+
+```python
+from composer import Trainer
+
+# use mixed precision during training
+trainer = Trainer(model=model,
+                  train_dataloader=train_dataloader,
+                  eval_dataloader=eval_dataloader,
+                  max_duration='160ep',
+                  device='gpu',
+                  precision='amp')
+```
+
+```{note}
+If the `precision` argument is not specified, the `Trainer` defaults to using `fp32`.
+```
+
+## Precision
+
+When discussing number formats, the precision generally refers to the number of total bits used to represent decimal digits (e.g., `fp32` represents numbers using 32 bits of computer memory).  Higher precision formats can represent a greater range of numbers and more decimal places compared to lower precision formats.  However, since lower precision formats utilize fewer bits, they occupy a smaller memory footprint.  Additionally, depending on the accelerator, lower precision formats can typically unlock more compute resulting in greater overall throughput.  Therefore it is often advantageous to use half-precision formats to accelerate training.
+
+For training, the most commonly used low precision formats are IEEE `fp16` and the Brain Floating Point `bf16` format.  While both formats utilize the same number of bits in memory, `bf16` offers a greater numerical range than `fp16` at the expense of being able to represent fewer decimal places, resulting in reduced sensitivity.
+
+Please see the following references for additioanl details on the common number formats:
+
+- `fp32`: [Single-precision Floating-point](https://en.wikipedia.org/wiki/Single-precision_floating-point_format)
+- `fp16`: [Half-precision Floating-point](https://en.wikipedia.org/wiki/Half-precision_floating-point_format)
+- `bf16`: [Brain Floating-point](https://en.wikipedia.org/wiki/Bfloat16_floating-point_format)
+
+### Mixed Precision
```

Comment:
I'm wondering if it would make sense to merge the AMP section below with this one....main reasoning is that AMP is how someone would interact with composer.

```suggestion
### Automatic Mixed Precision (AMP)
```